src/database/prismaClient.ts

```diff
@@ -78,9 +69,36 @@ export const prismaClient = new PrismaClient().$extends({
           },
         });
 
-        return {
-          error: false,
-        };
+        return { error: false };
+      },
+
+      async removeFromCurrentGame(
+        discordSnowflake: string
+      ): Promise<{ error: false } | { error: string }> {
+        try {
+          // Find the player by discord ID
+          const player = await prismaClient.player.findUnique({
+            where: { discordSnowflake },
+          });
+
+          if (!player) {
+            return { error: "Player not found." };
+          }
+
+          await prismaClient.gameParticipation.deleteMany({
+            where: { playerId: player.id },
+          });
+
+          return { error: false };
+        } catch (error) {
+          console.error(
+            `Error removing player ${discordSnowflake} from games:`,
+            error
+          );
+          return {
+            error: "Failed to remove the player from the current game.",
+          };
+        }
```

Comment:
we don't actually commit to GameParticipation during the game, but only after. i think this is unnecessary. also, a GameParticipation record is meant to be for past games, so we can view match history, it doesn't store anything about the current match, so removing all GameParticipations with a player id will just wipe their game history.

Reply:
ah ok lets remove